src/scenes/Species/SpeciesProjectsTable.tsx

```diff
@@ -181,9 +180,11 @@ export default function SpeciesProjectsTable({
           onSubmit={onRemoveHandler}
         />
       )}
-      {openedAddToProjectModal && selectableProjects && (
```

Comment:
Is it ok to remove this condition? _ selectableProjects_ ?

Reply:
I believe so, because `selectableProjects` is initialized as an empty array which is truthy. So this condition would never be false anyways.